src/structures/Guild.js

```diff
@@ -167,6 +167,14 @@ class Guild extends AnonymousGuild {
       this.large = Boolean(data.large);
     }
 
+    if ('premium_progress_bar_enabled' in data) {
```

Comment:
I still think this `in` check is useless since it's always provided according to the docs.. Thoughts @discordjs/the-big-4

Reply:
I followed what was done on the lines above. There are properties that are never missing according to the docs and still have that `in` check. I think that should be done in a separate PR if anything don't you think?